content/chainguard/chainguard-images/security-advisories/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "How to Use Chainguard Security Advisories"
+linktitle: "Security Advisories"
+type: "article"
+description: "Article outlining how one can explore and use the Security Advisories found on the Chainguard Image Directory."
+date: 2023-12-27T11:07:52+02:00
+lastmod: 2023-12-27T11:07:52+02:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 740
+toc: true
+---
+
+Say you use an image scanning tool like Grype or Docker Scout to determine what, if any, vulnerabilities are present within a given Chainguard Image. Chainguard Images typically report few or zero CVEs, but some Images — typically our Production Images, which consist of older versions of a given software and are therefore prone to vulnerabilities — may report more.
```

Comment:
Not sure we need to call out Production Images specifically. What do you think about this rewording? There also may be an opportunity to be more upfront about what this will be about.... I tried to do this by pulling the second paragraph up a bit

```suggestion
When you're checking to determine what vulnerabilities, if any, are present in a Chainguard Image, you may be using an image scanning tool like Grype or Docker Scout. Chainguard Images typically report few or zero CVEs, but when older versions of software are included in a given Image, a scanner may report more. Not all vulnerabilities are equal, and it may not be immediately clear whether a CVE found within a given Chainguard Image is likely to cause security issues in your software.
```

Reply:
I was struggling to not make Chainguard Images sound like they're riddled with CVEs, your wording is much better here. Thanks so much!